src/common/components/ButtonAdd/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
@muhadovic use one of ButtonAdd and ButtonClose. We don't need two components for same styling buttons.